parsers/GB.py

```diff
@@ -29,7 +29,7 @@ def fetch_price(
 
     url = ("https://www.rte-france.com/themes/swi/xml/power-market-data.xml")
 
-    response = r.get(url)
+    response = r.get(url, verify=False)
```

Comment:
Why are we not verifying SSL?

Reply:
I get this stacktrace without setting this parameter:

```2022-09-09 15:51:20,432 DEBUG    urllib3.connectionpool         Starting new HTTPS connection (1): www.rte-france.com:443
Traceback (most recent call last):
  File "/home/couderal/Perso/ElectricityMaps/electricitymap-contrib/.venv/lib/python3.10/site-packages/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/home/couderal/Perso/ElectricityMaps/electricitymap-contrib/.venv/lib/python3.10/site-packages/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/home/couderal/Perso/ElectricityMaps/electricitymap-contrib/.venv/lib/python3.10/site-packages/urllib3/connectionpool.py", line 1040, in _validate_conn
    conn.connect()
  File "/home/couderal/Perso/ElectricityMaps/electricitymap-contrib/.venv/lib/python3.10/site-packages/urllib3/connection.py", line 414, in connect
    self.sock = ssl_wrap_socket(
  File "/home/couderal/Perso/ElectricityMaps/electricitymap-contrib/.venv/lib/python3.10/site-packages/urllib3/util/ssl_.py", line 449, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "/home/couderal/Perso/ElectricityMaps/electricitymap-contrib/.venv/lib/python3.10/site-packages/urllib3/util/ssl_.py", line 493, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/usr/lib/python3.10/ssl.py", line 512, in wrap_socket
    return self.sslsocket_class._create(
  File "/usr/lib/python3.10/ssl.py", line 1070, in _create
    self.do_handshake()
  File "/usr/lib/python3.10/ssl.py", line 1341, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/couderal/Perso/ElectricityMaps/electricitymap-contrib/.venv/lib/python3.10/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/home/couderal/Perso/ElectricityMaps/electricitymap-contrib/.venv/lib/python3.10/site-packages/urllib3/connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "/home/couderal/Perso/ElectricityMaps/electricitymap-contrib/.venv/lib/python3.10/site-packages/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.rte-france.com', port=443): Max retries exceeded with url: /themes/swi/xml/power-market-data.xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/couderal/Perso/ElectricityMaps/electricitymap-contrib/.venv/lib/python3.10/site-packages/click/core.py", line 1134, in __call__
    return self.main(*args, **kwargs)
  File "/home/couderal/Perso/ElectricityMaps/electricitymap-contrib/.venv/lib/python3.10/site-packages/click/core.py", line 1059, in main
    rv = self.invoke(ctx)
  File "/home/couderal/Perso/ElectricityMaps/electricitymap-contrib/.venv/lib/python3.10/site-packages/click/core.py", line 1401, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/couderal/Perso/ElectricityMaps/electricitymap-contrib/.venv/lib/python3.10/site-packages/click/core.py", line 767, in invoke
    return __callback(*args, **kwargs)
  File "/home/couderal/Perso/ElectricityMaps/electricitymap-contrib/test_parser.py", line 60, in test_parser
    res = parser(*args, target_datetime=target_datetime, logger=getLogger(__name__))
  File "/home/couderal/Perso/ElectricityMaps/electricitymap-contrib/parsers/lib/config.py", line 15, in wrapped_f
    result = f(*args, **kwargs)
  File "/home/couderal/Perso/ElectricityMaps/electricitymap-contrib/parsers/GB.py", line 32, in fetch_price
    response = r.get(url)
  File "/home/couderal/Perso/ElectricityMaps/electricitymap-contrib/.venv/lib/python3.10/site-packages/requests/sessions.py", line 555, in get
    return self.request('GET', url, **kwargs)
  File "/home/couderal/Perso/ElectricityMaps/electricitymap-contrib/.venv/lib/python3.10/site-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/couderal/Perso/ElectricityMaps/electricitymap-contrib/.venv/lib/python3.10/site-packages/requests/sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "/home/couderal/Perso/ElectricityMaps/electricitymap-contrib/.venv/lib/python3.10/site-packages/requests/adapters.py", line 514, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='www.rte-france.com', port=443): Max retries exceeded with url: /themes/swi/xml/power-market-data.xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)')))

Reply:
Strange, I checked in the browser and it seemed to have a valid SSL cert.